Controllers/FileController.cs

```diff
@@ -23,27 +80,91 @@ public async Task<ActionResult> Post()
                 Directory.CreateDirectory(uploadPath);
             }
 
-            var path = Path.Combine(uploadPath, file.FileName);
+            foreach (var file in files)
+            {
+                var path = Path.Combine(uploadPath, file.FileName);
```

Comment:
Seems like a security issue. You probably want to sanitize file.FileName, otherwise, the uploader could upload a filename of "../../../doBadStuff.sh"